### ISSUE TITLE
Is there any way to control the thickness of the pen?

### ISSUE THREAD

**Issue by srimouli04** [excalidraw/excalidraw#7776]
Hi Maintainers,

When I try to reduce the size of the stroke its only fixed between 3 options and if I want a slimmer stroke line then I'll have to zoom out and then write is there any way around where I can be able to adjust the stroke thickness to my desired length in addition to the preset thickness lengths at any zoom level?

Thanks in advance

**Comment by iamsashika** [comment]
I agree. It would be extremely useful if there were more thickness options available, such as a range selection option, for the pen tool.
